src/main/java/frc/robot/controls/Driver.java

```diff
@@ -41,4 +46,48 @@ public Driver(Robot robot)
 	// METHODS //
 
 	// Add methods here which return values for various robot controls by reading the controllers.
+
+    /* =====================================
+    getThrottle() will return the current
+    value of the Y-Axis of the left stick on the controller
+    It will also handle adjusting
+    the reading according to the deadzone,
+    and according to the max throttle speed.
+    ===================================== */
+    public double getThrottle()
+    {
+        double throttleValue = 0;
+        
+        throttleValue = -dualshock.getY();
+        throttleValue *= Constants.MAX_THROTTLE_SPEED;
+
+        if (Math.abs(throttleValue) < Constants.DEADZONE)
+        {
+            return 0;
+        }
+
+        return throttleValue;
+    }
+
+    /* =====================================
+    getSteer() will return the current
+    value of the X-Axis of the right stick on the controller
+    joystick. It will also handle adjusting
+    the reading according to the deadzone,
+    and according to the max steer speed.
+    ===================================== */
+    public double getSteer()
+    {
+        double steerValue = 0;
+
+        steerValue = dualshock.getZ();
+        steerValue *= Constants.MAX_STEER_SPEED;
+
+        if (Math.abs(steerValue) < Constants.DEADZONE)
```

Comment:
Same thing here check previous comment for line 64, are you checking _controller deadzone_ or actual _motor deadzone_?